pkgs/by-name/ge/gerbolyze/package.nix

```diff
@@ -0,0 +1,92 @@
+{ lib
+, stdenv
+, pythonPackages
+, fetchFromGitHub
+, resvg
+}:
+
+let
+  pname = "gerbolyze";
+  version = "v3.1.7";
+  src = fetchFromGitHub {
+    owner = "jaseg";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-0iTelSlUJUafclRowwsUAoO44nc/AXaOKXnZKfKOIaE=";
```

Comment:
```suggestion
  version = "3.1.7";

  src = fetchFromGitHub {
    owner = "jaseg";
    repo = "gerbolyze";
    rev = "v${version}";
    hash = "sha256-0iTelSlUJUafclRowwsUAoO44nc/AXaOKXnZKfKOIaE=";
```

Reply:
Why would we remove `pname`? It can easily be inherited later and we don’t need to repeat the string. I’m indifferent to the change to `version`.

Reply:
pname should not be used here; it is not related to repo and can cause breakages down the track. This sort of thing is often advised against.

About the version, it's custom to use the numeric version number in the package version while the git tag is used in `rev`. From a style perspective I don't have much of an opinion on this but using a letter at the beginning of a package version could fool package databases such as repology.org and pkgs.org into thinking that we have a later version of the package than other package repos do.

Reply:
Done.